cypress/features/regression/loader.feature

```diff
@@ -1,36 +1,47 @@
-Feature: Loader component
+Feature: Loader default component
   I want to change Loader component properties
 
-  Background: Open Loader component page legacy spinner
-    Given I open "Loader" component page legacy spinner
+  Background: Open Loader default component page
+    Given I open "Loader" component page
 
   @positive
-  Scenario Outline: I set Loader component as to <as>
-    When I select as to "<as>"
-    Then Loader "border-top-color" color is set to "<border-top-color>"
-      And Loader "border-left-color" color is set to "0.7" gradient of "<border-top-color>" base color
-      And Loader "border-right-color" color is set to "0.3" gradient of "<border-top-color>" base color
+  Scenario Outline: I set Loader component size to <size>
+    When I select size to "<size>"
+    Then Loader width is set to "1059px" and height is set to "<height>"
```

Comment:
could we pass only an integer as a parameter without `px`? We could move `px` inside step

Reply:
fixed